code/src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+
+const Title = styled.h1`color: #bf3922;`;
+
+export const Header = () => {
+	const message = useSelector((state) => state.productStore.errorMessage);
+	const cameraOn = useSelector((state) => state.productStore.camera);
+	//show error message only when scanning
+	return (
+		<div>
+			<Title>Health App</Title>
+			{cameraOn ? message : ''}
```

Comment:
No need for a ternary here since there's no 'else' in this condition. You could use `&&` instead to only show the message when the camera is on:

```jsx
{cameraOn && message}
```